app/(website)/juguete/[slug]/page.tsx

```diff
@@ -11,16 +14,35 @@ interface Props {
 
 async function Page({ params: { slug } }: Props) {
   const toy = await client.fetch<Toy>(toySlugQuery, { slug });
-
+  const images = toy.images;
+  const description = toy.description[0];
   return (
     <section className="py-10">
-      <div className="mx-auto max-w-5xl">
-        <h2 className="mb-8 text-center text-2xl font-bold md:text-4xl">
+      <div className="mx-auto max-w-3xl">
+        <h4 className="mb-4 text-center">Juguetes</h4>
+        <h1 className="mb-8 text-center text-2xl font-bold md:text-4xl">
           {toy.name}
-        </h2>
-        <div>
-          <LoanRequestForm />
+        </h1>
+        <div className="relative mx-auto mb-10 lg:max-w-3xl">
+          <Carousel isFullWidth>
+            {images.map((image) => (
+              <Image
+                width={0}
+                height={0}
+                sizes="(max-width: 768px) 100vw"
```

Comment:
Later on, I may create a new issue to use the [`fill`](https://nextjs.org/docs/app/api-reference/components/image#fill) prop on these images.

Reply:
> Later on, I may create a new issue to use the [`fill`](https://nextjs.org/docs/app/api-reference/components/image#fill) prop on these images.

@TonyMckes Oh, trust me, bro, I tried `fill`, and it's not working... This Image component is a bit of a mess to work with. Seems like its conflicting in some way with Swiper.js